feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/domain/startCreateWallet/StartCreateWalletInteractor.kt

```diff
@@ -15,4 +17,8 @@ class RealStartCreateWalletInteractor(
     override suspend fun validateCanCreateBackup(): PreCreateValidationStatus {
         return cloudBackupService.validateCanCreateBackup()
     }
+
+    override suspend fun signInToCloud(): Result<Unit> {
```

Comment:
There is no point doing that for every screen since cloud backup service ensures that user is authenticated for every action in the service

Reply:
I added this function since we have an action "Sign in" in design and as we discussed this button should start authentication
![image](https://github.com/novasamatech/nova-wallet-android/assets/107959809/6b223283-db2a-47bb-a91c-62b319ef0d63)

Reply:
According to design - yes, we have this dialogue with the same action for every screen I added

Reply:
But according to the code there is no dialogue for other screens, it is done in view model init instead. That's why I am saying it is redundant - storage does the same automatically when doing any action